crates/turbo-tasks-memory/src/task.rs

```diff
@@ -762,7 +762,7 @@ impl Task {
         backend: &MemoryBackend,
         turbo_tasks: &dyn TurboTasksBackendApi,
     ) {
-        let mut queue = VecDeque::new();
+        let mut queue = VecDeque::with_capacity(0);
```

Comment:
Add a comment to explain that VecDeque allocates 7 elements by default. Otherwise I expect this to be accidentally changed back in the future.